packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -30,27 +30,38 @@ export const defaultLayouts = {
 	[ LAYOUT_TABLE ]: {
 		layout: {
 			primaryField: 'title',
+			combinedFields: [
+				{
+					id: 'post',
+					header: __( 'Title' ),
+					children: [ 'featured-image', 'title' ],
```

Comment:
I wonder if `Front Page` and `Posts Page` could now also be a field. Not sure if that needs to be filterable though.

Reply:
The features associated with making these fields don't seem particularly useful to me. Columns in particular would add weight to the table, especially as all records except one will be false. 

I appreciate it would make these pages a bit easier to locate, but not in a very ergonomic way. Add filter > is posts page > true isn't the quickest flow. Some alternatives we might consider could be:

1. Add a dedicated view for pages associated with settings. It would include the front page, posts page, privacy policy page (if set), products page, cart, checkout (if Woo is installed), etc.
2. Include the badge in when searching. IE if I search the All pages view for 'Front page', 'homepage', etc. then the static homepage should be listed even if the title doesn't contain those words.